build.gradle

```diff
@@ -165,19 +163,19 @@ task createKeystore(type: Exec) {
     commandArray.add('changeit')
     commandArray.add('-dname')
     commandArray.add("CN=localhost, OU=Engineering, O=Synopsys, C=US")
-    workingDir "${projectDir}/build/certs"
+    workingDir "${project.buildDir}/certs"
     commandLine commandArray
 }
 
 task createTruststore(type: Copy) {
     from "${System.getProperty('java.home')}/lib/security/"
     include 'cacerts'
     into "${project.buildDir}/certs/"
-    rename 'cacerts', "${project.buildDir}/certs/blackduck-alert.truststore"
+    rename 'cacerts', 'blackduck-alert.truststore'
```

Comment:
Is this only used in the runServer task?

Reply:
yes this is only used to create the trust store that alert will use when running.